versions.yaml

```diff
@@ -177,7 +177,7 @@ externals:
   attestation-agent:
     description: "Provide attested key unwrapping for image decryption"
     url: "https://github.com/confidential-containers/attestation-agent"
-    branch: "main"
+    version: "v0.1.0"
```

Comment:
Thanks for the update! Can I request we try using version `e930d362c24bad62b5fa7d5079a9dc912dd78fa4` at that is the current latest in main and I don't want us to be missing function going back to the old tag?

Reply:
Sure, good catch :), updated to the hash.